src/core/weightedGraph.test.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+
+import * as Weights from "./weights";
+import {Graph} from "./graph";
+import * as WeightedGraph from "./weightedGraph";
+import * as GraphTest from "./graphTestUtil";
+
+describe("core/weightedGraph", () => {
+  function expectEqual(wg1, wg2) {
+    expect(wg1.graph.equals(wg2.graph)).toBe(true);
+    expect(wg1.weights).toEqual(wg2.weights);
+  }
+  it("empty produces an empty WeightedGraph", () => {
+    const {weights, graph} = WeightedGraph.empty();
+    expect(graph.equals(new Graph())).toBe(true);
+    expect(weights).toEqual(Weights.empty());
+  });
+  describe("toJSON/fromJSON", () => {
+    it("works for an empty WeightedGraph", () => {
+      const wg = WeightedGraph.empty();
+      const wgJSON = WeightedGraph.toJSON(wg);
+      const wg_ = WeightedGraph.fromJSON(wgJSON);
+      expectEqual(wg, wg_);
+    });
+    it("works for a non-empty WeightedGraph", () => {
+      const node = GraphTest.node("foo");
+      const graph = new Graph().addNode(node);
+      const weights = Weights.empty();
+      weights.nodeWeights.set(node.address, 5);
+      const wg = {graph, weights};
+      const wgJSON = WeightedGraph.toJSON(wg);
+      const wg_ = WeightedGraph.fromJSON(wgJSON);
+      expectEqual(wg, wg_);
+    });
+  });
+  it("merge works on nontrivial graph", () => {
```

Comment:
Nit: same as above
```
describe "merge"
  it "works on ____"
```

Reply:
yeah, in this case it's easier for me to imagine adding more tests, so i'm more open to adding a describe block.

i'll even do it for empty too, i don't think it's very necessary, but it is more consistent.